extensions/http-curl/tests/unit/InvokeHTTPTests.cpp

```diff
@@ -270,19 +276,21 @@ TEST_CASE("HTTPTestsWithResourceClaimPOST", "[httptest1]") {
     REQUIRE(provEventRecord->getComponentType() == listenhttp->getName());
   }
   std::shared_ptr<core::FlowFile> ffr = session2->get();
-  REQUIRE(true == LogTestController::getInstance().contains("exiting because method is POST"));
+  REQUIRE(true == LogTestController::getInstance().contains("Exiting because method is POST"));
   LogTestController::getInstance().reset();
 }
 
 TEST_CASE("HTTPTestsPostNoResourceClaim", "[httptest1]") {
   TestController testController;
-  LogTestController::getInstance().setInfo<org::apache::nifi::minifi::processors::InvokeHTTP>();
-  LogTestController::getInstance().setInfo<org::apache::nifi::minifi::processors::ListenHTTP>();
+  LogTestController::getInstance().setDebug<org::apache::nifi::minifi::processors::ListenHTTP>();
```

Comment:
Why is it better to set it to Debug level from info level?

Reply:
In the test we are checking for "Exiting because method is POST" log message which is a debug log, so we have to modify the log level to be able to assert on that. It was probably an info level log previously and changed later.

Reply:
Ahan it makes sense now. Thanks :)